uli_init/simulate.py

```diff
@@ -349,432 +385,38 @@ def anneal(
                 dynamic=["momentum"]
             )
 
-            for kT in schedule:  # Start iterating through annealing steps
-                print(f"Running @ Temp = {kT} kT")
+            if pressure:
+                try:
+                    integrator.disable()
+                except:
+                    pass
+                integrator = hoomd.md.integrate.npt(
+                        group=_all,
+                        tau=self.tau_kt,
+                        tauP=self.tau_p,
+                        P=pressure,
+                        kT=1
+                        )
+            elif not pressure:
+                try:
+                    integrator.__getattribute__
+                except:
```

Comment:
```suggestion
                    integrator
                except NameError:
```
same as above comments